### ISSUE TITLE
[Feature Request] Force enable zoom

### ISSUE THREAD

**Issue by Uncle-Squishy** [tachiyomiorg/tachiyomi#4186]
## Why/User Benefit/User Problem
Some useres (like me) don't have the best eyesight and zoom in on areas of theamga or the whole page to read it easier. For some reason, there are various series that you currently can't zoom intoaling for a difficult reading experience. 

## What/Requirements
Put a toggle in the display settings to allow forced zoom on series where it wasn't working. Then you just pinch to zoom as usual

**Comment by CarlosEsco** [comment]
Can you give an example of something that you cant zoom?  There is nothing in the code that is preventing zoom per a series

**Comment by Uncle-Squishy** [comment]
I can't remember the name of the series at the moment as I've been on a bit
of a binge. I do know the chapter was from Noname scans. I think it was the
combination of using a tablet to read a low resolution chapter. So far it
has been the only chapter to be an issue.

**Comment by pogo1975** [comment]
I can second to that, example for instance source readcomicsonline comics iron mam 2020 (1995) on my tablet in landscape mode icant zoom in to resize image width it centers on screen
![Screenshot_20210207-195033_Tachiyomi](https://user-images.githubusercontent.com/8178071/107156292-d2005e00-697d-11eb-9ce2-4403cbedeb83.png)

**Comment by gregorioepc** [comment]
Please unlock/allow free zooming as it is hard to read some panels with small texts, especially for readers with bad eyesight like us. I do use the 'continuous vertical' read mode as i found that it allows a bit of extra zoom levels compared to the normal 'left to right' but still find it difficult depending on the circumstance (small texts, bad fonts, etc.). One piece double page spreads for example, i have to screencap panels and use zoom with my phone. 

Thank you very much!

**Comment by FavroYorseif** [comment]
Hello, 
First, I would like to apologize in advance if I make a mistake here. I am new. I don't know if issues regarding tachij2k is allowed here, but i don't know where else to comment. I am also having the same issue as this thread (preety bad eyesight). Unlike before (right before I installed the newest update, which is a few minutes ago), I am unable to zoom in indefinitely long stripped pages while on vertical read mode. I can still zoom it in but not to the extent that it's readable. Also it depends on the length of the strips. 

I appreciate all the help I can get. Thank you very much! 

P. S. If you allow me to be a bit thick skinned. Can someone, please, tell me how to make it so I am able view long strips on webtoon mode. All I see is a blackscreen, tried waiting if it is only loading, but nothing happens. Again, Thank you!

**Comment by saud-97** [comment]
Hi, we are currently developing a solution to this issue https://github.com/tachiyomiorg/tachiyomi/pull/7029. The issue happens because images that are extremely long cannot be loaded properly inside tachiyomi reader. Soon this issue will get fixed.

**Comment by FavroYorseif** [comment]
> Hi, we are currently developing a solution to this issue #7029. The issue happens because images that are extremely long cannot be loaded properly inside tachiyomi reader. Soon this issue will get fixed.

Wow! That's great news. Thank you!

**Comment by FavroYorseif** [comment]
Hi, it's me, again. I tried to skim through issue #7029. Correct me if I'm wrong, does that mean it'll only work on downloaded chapters? If so, is there a possibility of it working in, uhm, undownloaded (haha)? I'm sorry, I don't really understand codes or coding stuff, so I hope that I don't seem like undermining what you're doing. I appreciate it really.

**Comment by saud-97** [comment]
Yes for now it's only for downloaded chapters. Regarding the reading without downloading I'm not sure but you may request it as a feature once this fix is implemented.

**Comment by arkon** [comment]
Please use the relevant issues to discuss things.

- Large images not loading on weaker devices in webtoon/continuous vertical (for which the split downloaded images PR is relevant): #1908
- Can't zoom in far enough on webtoon/continuous vertical: #6834
